packages/flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -47,17 +63,31 @@ class Selection {
   }
 }
 
+/// {@template _base_selection}
+/// Base class for all FireAtlas selections.
```

Comment:
does this represent a region of the image? maybe we should point that out here

Reply:
also would be good to mention what children exists and how they differ

Reply:
```suggestion
/// Base class for all FireAtlas selections, containing information for a section of the image.
```